### DISCUSSION TITLE
Intermittent build issue: "SyntaxError: Named export not found" (works on redeploy)

### DISCUSSION THREAD

**Discussion by kristiegiles** [vercel/community#7050] [Help]
### Summary

I started getting this error intermittently on the first build (using turborepo default configuration). It always works fine on redeploy and works fine locally.  I'm not sure where to start with debugging this – does anyone have any ideas? 

<img width="1486" alt="Screenshot 2024-05-30 at 5 01 36 PM" src="https://github.com/vercel/community/assets/25679998/fae92fc4-d675-47d4-960b-7fa98c7b84e4">
 

### Steps to Reproduce

I am using turborepo and am on the Vercel Pro plan with 2 concurrent builds. Often, 4+ builds are triggered at once and some of them fail on the initial build and only work after manually redeploying.

**Comment by kristajg-vercel** [comment]
Hi, have you tried changing your import to as follows (based on logs output)? Looks like a differentiation in how the JS module is imported? I also see eventemitter3 docs recommending to import it pre-es6 manner

CommonJS modules can always be imported via default export, for example using:

```
import pkg from 'eventemitter3'
const { EventEmitter } = pkg
```

**Reply by kristiegiles** [reply]
The import is within a third-party library, so I can't easily change it. Why would it work locally and on redeploy but only fail intermittently if the syntax is incorrect?

**Reply by amyegan** [reply]
If it's working on redeploy, that could be a build cache issue. Does it work on the first deployment if you [disable the build cache](https://github.com/vercel/vercel/discussions/8057#discussioncomment-3057196)?

**Reply by kristiegiles** [reply]
It works on the first deploy sometimes even without remote caching disabled. It is an intermittent issue, so I can't reproduce super reliably. I think I have also had it happen with remote caching disabled, though, and disabling remote caching makes things slow.

[a comment by an automated account is not shown]
